### ISSUE TITLE
White shear lines on masked images

### ISSUE THREAD

**Issue by lucastucious** [naturalcrit/homebrewery#3656]
### Renderer

v3

### Browser

Chrome

### Operating System

Windows

### What happened?

See issue on reddit : https://www.reddit.com/r/homebrewery/comments/1es0me9/white_shear_lines_appear_on_my_images_when_i/

Basically, there is a some lines appaearing on the export in masked images

**Comment by lucastucious** [comment]
![image](https://github.com/user-attachments/assets/43a36226-1e6a-432c-8e82-51e5603583e6)
Here is some extreme values to show the issue. I've tried many things in css to fix this but it seems like it's a bug from the chrome printing engine

**Comment by G-Ambatte** [comment]
Correct, it actually appears worse in the Print Preview than it is in the actual output PDF - it's still present, just far less visually noticeable.

As best I can tell, it has to do with the `transform` which rotates the mask leaving an unclean edge at the point that it tiles. There isn't much we can do about it directly, as it is a Chrome bug.

The workaround at the moment is to adjust the scale parameters to move the white lines out of the visible area.... Unfortunately there is also a bug with mask scaling that causes image skewing (#3636) but the fix has been created, it just needs to be approved and hopefully will be in the next release to go live on the site.

...also I'm certain this has been reported before, although I can't find an open issue right now.

**Comment by 5e-Cleric** [comment]
Chromium more than chrome issue, since it also happens in Opera GX, but still horrible to deal with.

**Comment by dbolack-ab** [comment]
Still ongoing?

**Comment by lucastucious** [comment]
> Still ongoing?

Yes. Still an issue on the pdf preview and the printed pdf.

Here is an example snippet :

```md
{{imageMaskEdge1,--offset:-3%,--rotation:0,bottom:49%
  ![](https://i.redd.it/0mq2tx1s3up71.jpg){left:-2px}
}}
```

**Comment by 5e-Cleric** [comment]
Nothing we can do to fix this, its an aliasing bug on chromium's renderer, can't find the specific issue right now.
